bootstrap/bootstrap.sh

```diff
@@ -4,12 +4,18 @@ HERE=$(dirname $(readlink -f $0))
 set -ex
 
 #TODO wish we could bundle this in a single apply, but
-# - 'helm template' is not emitting the labels helm needs to upgrade it later 
+# - 'helm template' is not emitting the labels helm needs to upgrade it later
 # - helm dependencies are just terrible
 
 kubectl create namespace foundations
 kubectl apply -f ${HERE}/user-skeleton.yaml
+
+# helm repo add cilium https://helm.cilium.io/
 helm install --namespace foundations cilium cilium/cilium --version 1.15.7 --values ${HERE}/cilium-values.yaml
-flux install --namespace foundations
-helm install --namespace foundations foundations  ${HERE}/../charts/foundations --version 0.1.0
 
+kubectl apply -k ${HERE}/../flux
```

Comment:
this doesnt look right. thats without kustomize isnt it? so wrong namespace

Reply:
That's kustomize; `-k` is kustomize. The file embeds the namespace (I could extract it too)